datafusion/core/src/physical_planner.rs

```diff
@@ -1798,18 +1798,14 @@ impl DefaultPhysicalPlanner {
                         }
                     }
                     Err(err) => {
-                        return Ok(Some(Arc::new(ExplainExec::new(
-                            SchemaRef::new(Schema::new(vec![arrow_schema::Field::new(
-                                "Err",
-                                arrow_schema::DataType::Utf8,
-                                false,
-                            )])),
-                            vec![StringifiedPlan::new(FinalLogicalPlan, err.to_string())],
-                            e.verbose,
-                        ))))
+                        stringified_plans.push(StringifiedPlan::new(
```

Comment:
this is the core difference -- rather than using a special `ExplainExec` instead use a new "plan type" that isn't filtered out of the normal explain